File: Looping/Look Behind to See Forward/Other Lang/C/LookBehindTwo.c
#include <stdio.h>

void main() {
	int prev_num2 = 0;
	int prev_num1 = 1;
	int number = 0;
	printf("%d ", prev_num2);
	printf("%d ", prev_num1);
	for (int i=0; i < 8; i++) {
		number = prev_num2 + prev_num1;
		prev_num2 = prev_num1;
		prev_num1 = number;
		printf("%d ", number);
	}
	printf("\n");
}

File: Looping/Look Behind to See Forward/Other Lang/Python/LookBehindTwo.py
def main():
	prev_num2 = 0
	prev_num1 = 1
	print(''.join([str(prev_num2), " "]), end="")
	print(''.join([str(prev_num1), " "]), end="")
	for i in range (0,8):
		number = prev_num2 + prev_num1
		prev_num2 = prev_num1
		prev_num1 = number
		print(''.join([str(number), " "]), end="")
	print("")

main()
